source/com.google.crypto.tink/tink-android/Transforms/Metadata.xml

```diff
@@ -666,4 +666,175 @@
         Xamarin.Google.Crypto.Tink.Parameters
     </attr>
 
+
+    <attr
+        path="/api/package[@name='com.google.crypto.tink.daead']/class[@name='AesSivKey']/method[@name='getParameters' and count(parameter)=0]"
+        name="managedReturn"
+        >
+        Xamarin.Google.Crypto.Tink.Parameters
+    </attr>
+    <attr
+        path="/api/package[@name='com.google.crypto.tink.daead']/class[@name='DeterministicAeadKey']/method[@name='getParameters' and count(parameter)=0]"
+        name="managedReturn"
+        >
+        Xamarin.Google.Crypto.Tink.Parameters
+    </attr>
+
+    <add-node
```

Comment:
I don't understand or like the use of `<add-node/>` here.  Adding a `<method/>` is saying "The Java side has a method with *this* signature", which could potentially result in Java `NoSuchMethodError`.

For example, the `<method/>` below tells `generator` that the method `getParameters()Lcom/google/crypto/tink/Parameters;` exists, but that method doesn't exist: https://github.com/google/tink/blob/9bc2667963e20eb42611b7581e570f0dddf65a2b/java_src/src/main/java/com/google/crypto/tink/streamingaead/AesGcmHkdfStreamingKey.java#L65

The method that *actually* exists is `getParameters()Lcom/google/crypto/tink/streamingaead/AesGcmHkdfStreamingParameters;`, and trying to invoke the latter with the former *could* fail.

Why is `<add-node/>` needed here and elsewhere?